madgraph/interface/amcatnlo_run_interface.py

```diff
@@ -5272,6 +5272,74 @@ def copy_lep_densities(self, name, sourcedir):
             files.cp(os.path.join(lep_d_path, filename), pdf_path)
 
 
+    def link_and_copy_epdf(self, pdlabel, lhaid, libdir):
```

Comment:
why this change here? unrelated to the FD gauge, right?

Reply:
Looks like this is a git error (or when solving a conflict), since now this function is define twice within this file (line 5207 and 5275)... Thanks for spotting this.